lib/modules/dosomething/dosomething_taxonomy/dosomething_taxonomy.module

```diff
@@ -110,15 +110,21 @@ function dosomething_taxonomy_preprocess_taxonomy_term(&$vars) {
 
 /**
  * Returns array of node nid's with the given Term $tid.
- * @deprecated Use taxonomy_select_nodes instead.
  *
  * @param int $tid
  *   A taxonomy term tid.
  *
  * @return array
  */
 function dosomething_taxonomy_get_campaigns($tid) {
-  return taxonomy_select_nodes($tid, FALSE);
+  $nodes = node_load_multiple(taxonomy_select_nodes($tid, FALSE));
```

Comment:
@angaither where do we add a type? The docs list the conditions as deprecated https://api.drupal.org/api/drupal/modules%21node%21node.module/function/node_load_multiple/7.x